app/components/TrustedSourceList.js

```diff
@@ -27,41 +32,35 @@ export const TrustedSourceList = ({
           style={styles.authorityFilter}
           onPress={() => toggleFilterAuthoritesByGPSHistory()}>
           <Typography style={styles.authorityFilterText} use={'body2'}>
-            {languages.t('label.filter_authorities_by_gps_history')}
+            {t('label.filter_authorities_by_gps_history')}
           </Typography>
           <Switch
             onValueChange={val => filterAuthoritesByGPSHistory({ val })}
             value={isAuthorityFilterActive}
           />
         </TouchableOpacity>
       )}
-      {authoritiesList === undefined
-        ? null
-        : authoritiesList.map(item => {
-            let name = Object.keys(item)[0];
-            let key = authoritiesList.indexOf(item);
+      {availableAuthorities &&
+        availableAuthorities.map(item => {
```

Comment:
create a unit test for this to show that it renders the correct lists depending on availableAuthorities and selectedAuthorities.